docs/user-guide/installandconfig.md

```diff
@@ -17,6 +17,18 @@ zFS is a UNIX file system where Zowe runtime files and folders are installed. Zo
 
 For more information about USS, see [Addressing UNIX System Servies (USS) Requirements](./configure-uss).
 
+:::note
+Zowe runs in USS, it makes use of shell scripts and TCP/IP sockets. The amount of open I/O handles and the sizes that you may see are the result of normal use of the above resources. If you do not observe any other malfunction, the number of open files should not be a concern.
```

Comment:
The sizes of what? Where would malfunction be detected? What type of malfunction...insufficient available memory?

Reply:
... like reaching the socket limits or limit on the amount of files in the specific directory ...